ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoiceNotifier.java

```
@@ -218,6 +221,7 @@ private void internalAttestationsDue(final UInt64 slot) {
     updatePayloadAttributes(slot.plus(1));
   }
 
+  @SuppressWarnings("FutureReturnValueIgnored")
   private void sendForkChoiceUpdated() {
     forkChoiceUpdateData.send(executionEngineChannel);
   }
```

Ignoring the return value here is definitely wrong - we'd silently ignore any exceptions (which might include the queue being full).  Could potentially use `.reportExceptions()` here if we shouldn't ever get an error from this.

---

yeah, thought the same going bed...